packages/x-charts/src/LineChart/MarkElement.tsx

```diff
@@ -54,13 +53,11 @@ const useUtilityClasses = (ownerState: MarkElementOwnerState) => {
   return composeClasses(slots, getMarkElementUtilityClass, classes);
 };
 
-const MarkElementPath = styled('path', {
+const MarkElementPath = styled(animated.path, {
```

Comment:
Total nitpick, but have you considered going for a faster transition on mark elements to avoid them lagging as compared to the line/area when doing fast data updates? 🤔 🙈 

https://github.com/mui/mui-x/assets/4941090/a8789c5c-e9b1-4b1b-84bf-68c21d342a11

Reply:
No, because this demo is out of real world consideration.

- If you need that fast refresh: you disable the animation
- If you don't update more than once every second you don't see the bug

Reply:
Fair point. Agreed, this is a total edge-case. 👌